ThingIFSDK/ThingIFSDK/IoTOperations/IoTRequestOperation.swift

```diff
@@ -33,7 +33,13 @@ func buildDefaultRequest(method : HTTPMethod,urlString: String,requestHeaderDict
     kiiVerboseLog("Request Method: \(method)")
     kiiVerboseLog("Request Header: \(requestHeaderDict)")
 
-    let defaultRequest = DefaultRequest(method: method, urlString: urlString, requestHeaderDict: requestHeaderDict, requestBodyData: requestBodyData, responseBodySerializer: { (responseBodyData) -> NSDictionary? in
+    // Add X-Kii-SDK header.
+    var modifiedHeaderDict:Dictionary<String, String> = requestHeaderDict;
```

Comment:
please remove semicolon.
redefinition of Dictionary<String, String> is not necessary, below will be sufficient : 
```swift
var modifiedHeaderDict = requestHeaderDict 
```